web/src/click_handlers.js

```diff
@@ -659,6 +659,11 @@ export function initialize() {
             return;
         }
 
+        if ($target.is("#send_later i")) {
+            e.stopPropagation();
+            return;
+        }
```

Comment:
Yeah, makes sense.

Reply:
Hmmm, looks like we do need it, added a comment there to explain it.

```
            // Since the click for this is handled by tippyjs, we cannot add stopPropagation
            // there without adding a special click event handler to show the popover,
            // so it is better just do it here.
````